### ISSUE TITLE
NAT Loopback is disabled when Software Fast NAT path is enabled

### ISSUE THREAD

**Issue by GoogleCodeExporter** [padavanfirmware/rt-n56u#298]
```
What steps will reproduce the problem?
1. Enable NAT Loopback in firewall settings
2. Select Fast NAT Pat in Software NAT of WAN settings
3. Use a DDNS account
4. On a PC within your LAN, try to telnet into the same PC using the DDNS 
(external IP). 

What is the expected output? What do you see instead?
Expected Output : To be able to connect to internal devices using the external 
IP.
Instead : Unable to connect

What version of the product are you using? On what operating system?
Firmware : 1.1.1.8f-b6

Please describe the problem as detailed as it's possible.
Basically NAT Loopback does not work when Software Fast NAT is enabled. 
Changing Software NAT back to Default NAT works.
```

Original issue reported on code.google.com by `unfur...@gmail.com` on 27 May 2012 at 10:04
